### PR TITLE
Fix low cost template

### PR DESCRIPTION
**Pull request by sonofagl1tch**
*Description of changes:*

hardcoded smallest instance type that the bastion host can use. this fixes the 404 issue

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsDetonationLab.template [hunk 1]
```diff
@@ -3061,6 +3017,48 @@
           "Ref": "wazuhUser"
         }
       }
+    },
+    "KinesisToLambda": {
```
Comment (mgmacias95) [inline awsDetonationLab.template]: Is this change necessary in this PR?

Comment (sonofagl1tch) [reply awsDetonationLab.template]: I am pretty sure this change is just the lines being moved due to the actual changes

**Review by mgmacias95 — APPROVED** [review]
